README.md

```diff
@@ -9,10 +9,144 @@
+| `stability` | `stability` | | | |
+| `homepage` | `homepage` | If `github` given, `<repo>#readme` | | |
+| `bug-reports` | `bug-reports` | If `github` given, `<repo>/issues` | | |
+| `author` | `author` | | May be a list | |
+| `maintainer` | `maintainer` | | May be a list | |
+| `copyright` | `copyright`          |                                    | May be a list         |
+| `license` | `license` | | | |
+| `license-file` | `license-file` | `LICENSE` if file exists | | |
+| `tested-with` | `tested-with` | | | |
+| | `build-type` | `Simple` | | |
+| | `cabal-version` | `>= 1.10` or `>= 1.21` | `>= 1.21` if library component has `reexported-modules` field | |
+| `extra-source-files` | `extra-source-files` | | Accepts glob patterns | |
+| `data-files` | `data-files` | | Accepts glob patterns | |
+| `github` | `source-repository head` | | Accepts `user/repo` or `user/repo/subdir` | `github: foo/bar`
+| `git`    | `source-repository head` | | No effect if `github` given | `git: https://my.repo.com/foo` |
+| `flags`  | `flag <name>` | | Map from flag name to flag (see **flags** section below) | |
+| `library` | `library` | | See **library fields** section below | |
+| `executables` | `executable <name>` | | Map from executable name to executable (see **executable fields** section below) | |
+| `tests` | `test-suite <name>` | | Map from test name to test (see **test fields** section below) | |
+| `benchmarks` | `benchmark <name>` | | Map from benchmark name to benchmark (see **benchmark fields** section below) | |
+
+#### Global top-level fields
+
+These fields are merged with all library, executable, test, and benchmark components.
+
+| Hpack | Cabal | Default | Notes |
+| --- | --- | --- | --- | --- |
+| `source-dirs` | `hs-source-dirs` | | |
+| `default-extensions` | `default-extensions` | | |
+| `other-extension` | `other-extensions` | | |
+| `ghc-options` | `ghc-options` | | |
+| `ghc-prof-options` | `ghc-prof-options` | | |
+| `cpp-options` | `cpp-options` | | |
+| `c-sources` | `c-sources` | | |
+| `extra-lib-dirs` | `extra-lib-dirs` | | |
+| `extra-libraries` | `extra-libraries` | | |
+| `include-dirs` | `include-dirs` | | |
+| `install-includes` | `install-includes` | | |
+| `ld-options` | `ld-options` | | |
+| `buildable` | `buildable` | | May be overridden by later stanza |
+| `dependencies` | `build-depends` | | |
+| `when` | | | Accepts a list of conditionals (see **conditionals** section below) |
+
+#### Library fields
+
+| Hpack | Cabal | Default | Notes |
+| --- | --- | --- | --- |
+| `exposed` | `exposed` | | |
+| `exposed-modules` | `exposed-modules` | All modules in `source-dirs` less `other-modules` | |
+| `other-modules` | `other-modules` | All modules in `source-dirs` less `exposed-modules` | |
+| `reexported-modules` | `reexported-modules` | | |
+| | `default-language` | `Haskell2010` | |
+
+#### Executable fields
+
+| Hpack | Cabal | Default | Notes |
+| --- | --- | --- | --- |
+| `main-is` | `main-is` | | |
```

Comment:
Should `main-is` be changed to `main` in the Hpack column for the executable, test, and benchmark sections?